integration/simulation/transaction_injector.go

```diff
@@ -284,37 +310,49 @@ func (m *TransactionInjector) rndL2NodeClient() *obscuroclient.Client {
 	return m.l2Clients[rand.Intn(len(m.l2Clients))] //nolint:gosec
 }
 
-// NewL2Transfer creates an enclave.L2Tx of type enclave.TransferTx
-func NewL2Transfer(from common.Address, dest common.Address, amount uint64) types.TxData {
-	txData := core.L2TxData{Type: core.TransferTx, From: from, To: dest, Amount: amount}
-	return NewL2Tx(txData)
+func NewObscuroTransferTx(from wallet.Wallet, dest common.Address, amount uint64, client *obscuroclient.Client) types.TxData {
```

Comment:
Can comments be added to the public methods please ?

Reply:
oops, these shouldn't be public.